### DISCUSSION TITLE
How to reduce the size of a reproduction?

### DISCUSSION THREAD

**Discussion by harry643** [squidfunk/mkdocs-material#4809] [Q&A] [answered]
I'm trying to submit a bug report and my minimal reproduction zip is 26MB. The Github limit is 25MB. This is a completely new mkdocs project with a venv containing only the mkdocs-material Python package.

Please can I have some guidance on this?

**Comment by squidfunk** [comment, marked as answer]
Thanks for asking. The `venv` must not be included in the zip. As noted in our new [reproduction guide](https://squidfunk.github.io/mkdocs-material/bug-report/reproduction/#creating-a-zip-file), please use the `info` plugin, which will only include the necessary files.

**Reply by harry643** [reply]
Ok thank you. I assumed that plugin was optional!

**Reply by squidfunk** [reply]
No, as our [new issue template](https://github.com/squidfunk/mkdocs-material/blob/master/.github/ISSUE_TEMPLATE/bug.yml) mentions, it is mandatory:

> Please create a minimal reproduction for the bug. First, read our [reproduction guide](https://squidfunk.github.io/mkdocs-material/bug-report/reproduction/) that explains the necessary steps, then use the [built-in info plugin](https://squidfunk.github.io/mkdocs-material/bug-report/reproduction/#creating-a-zip-file) (mandatory) to create a self-contained .zip file with the reproduction. Without a reproduction, it can be challenging to fix the bug. [More](https://squidfunk.github.io/mkdocs-material/bug-report/#reproduction)
